eks-distro-base/make-tests/expected/standard/images/no-deps-presubmit

```diff
@@ -1,8 +1,8 @@
 
 # This file was generated by running `make run-make-tests` and `make update-make-tests-expected`. DO NOT EDIT
 
-/override/root/../scripts/buildkit_check.sh
+/Users/sjparekh/Documents/local_development/github/eks-distro-build-tooling/eks-distro-base/../scripts/buildkit_check.sh
```

Comment:
this is pointing to your local path, could you check this?

Reply:
Updated